### DISCUSSION TITLE
Cannot compile cross-compiler.

### DISCUSSION THREAD

**Discussion by zap8600** [community/community#31380] [Codespaces]
I was building a cross-compiler for [my OS](https://github.com/zap8600/CCOS-new). When I got to gcc, I got an error saying `Killed signal terminated program cc1plus`. I haven't gotten this error outside of codespaces, so I thought I could get some help here. Thanks in advance.

**Comment by fara-nak** [comment]
hi @zap8600 sorry you had difficulties using codespaces, would you please share some logs so we can see more info of the issue? you can do that by going to command pallet and click `Codespaces: Export Logs`

**Reply by fara-nak** [reply]
also could you please tell us the repro steps?

**Reply by zap8600** [reply]
I got gcc to build this time. Now, I have another issue. 
After a while of being inactive, the codespace resets. The cross-compiler will be deleted, and I'll have to build it again. 
Is there a way around this? My cross-compiler is installed at `~/opt/cross`. I could build it in the Github repository, but I wouldn't like to unless necessary.

**Reply by greggroth** [reply]
Have you tried using a higher idle timeout for your codespace? You can either [set a higher default timeout under Codespace settings](https://docs.github.com/en/codespaces/customizing-your-codespace/setting-your-timeout-period-for-github-codespaces) or use the `--idle-timeout` flag when creating a codespace through the `gh` cli.

**Reply by zap8600** [reply]
I will try this. 
The only thing I have with this is that the timer will still run out and the compiler will be wiped. 
Is there a way to prevent it from resetting?
